.ci/install_kubernetes.sh

```diff
@@ -39,9 +34,7 @@ chronic sudo -E apt install kubernetes-cni=0.6.0-00
 
 chronic sudo -E apt install --allow-downgrades -y kubelet="$kubernetes_version" kubeadm="$kubernetes_version" kubectl="$kubernetes_version"
 
-if [ "${use_runtime_class}"  == true ]; then
-	kubelet_systemd_file="/etc/systemd/system/kubelet.service.d/10-kubeadm.conf"
-	feature_gate="--feature-gates RuntimeClass=true"
-	echo "Configure Kubelet service to enable RuntimeClass feature"
-	sudo sed -i "s/ExecStart=\/.*$/& $feature_gate/" "$kubelet_systemd_file"
-fi
+kubelet_systemd_file="/etc/systemd/system/kubelet.service.d/10-kubeadm.conf"
+feature_gate="--feature-gates RuntimeClass=true"
```

Comment:
With 1.14 RuntimeClass is beta. So it is enabled by default. We can remove the feature-gate everywhere

Reply:
krsna1729 changes applied